Formula/odo-dev.rb

```diff
@@ -0,0 +1,34 @@
+class OdoDev < Formula
+  desc "Developer-focused CLI for Kubernetes and OpenShift"
+  homepage "https://odo.dev"
+
+  url "https://github.com/redhat-developer/odo.git",
+    tag:      "v2.5.0",
+    revision: "724f16e689545dd4a81671da3e116a33df4832d3"
+
+  license "Apache-2.0"
+  head "https://github.com/redhat-developer/odo.git", branch: "main"
+
+  depends_on "go" => :build
+  conflicts_with "odo", because: "odo also ships 'odo' binary"
+
+  def install
+    system "make", "bin"
+    bin.install "odo"
```

Comment:
Is there no `make install`?

Reply:
There is `make install` but that uses `go install` to install binary to `$GOPATH`. I don't think it should be used here.